src/app/_components/page-assets/contact-section.tsx

```diff
@@ -3,20 +3,20 @@ import Section from "./section";
 
 export default function ContactSection({
   title,
-  bg,
+  className,
 }: {
   title: string;
-  bg: string;
+  className: string;
```

Comment:
I don't think this component should have a className or a custom title.

Is it supposed to be a presentational component with multiple sections of customizable design (title, description, primary button)? As I believe that the Section Component should be highly customizable, and the contact section should really just be the "configured" component. 

Take a look at [Shopify's React Components](https://polaris.shopify.com/components/layout-and-structure/card?example=card-with-multiple-sections). As you can see here the base component has a lot of customizability to allow a developer to extend and craft it into a purposeful specific component like a contact section. If this "configured" component is being re-used, then they should simply export the component with its configurations (no customizability), otherwise they should "reconfigure" from the Section Component for that specific page.